firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -117,6 +118,7 @@ public ConfigFetchHandler(
     this.frcBackendApiClient = frcBackendApiClient;
     this.frcMetadata = frcMetadata;
     this.customHttpHeaders = customHttpHeaders;
+    this.lastTemplateVersion = getTemplateVersionFromDisk();
```

Comment:
I'd avoid doing work (esp reading from disk) in the constructor - if it's required we could pass it in. It looks like the instance variable is used to get the latest fetched version in ConfigAutoFetch (rather than being required in this class), though. Does it need to be initialized from a saved value in any event?

Reply:
Updated to only update when the accessor method is called. And I keep the templateVersion in fetch b/c its more of a part of fetch and the fetch response then anything and it isn't entirely a Realtime change. Also, there are other ways that fetch can be invoked instead of Realtime, in this cases we would also want to keep updating the templateVersion.